notebooks/deepdive/custom/ucaip_customjob.ipynb

```diff
@@ -0,0 +1,1857 @@
+{
```

Comment:
<p>Don't use the internal link (pantheon.corp.google.com) in a public tutorial. Swap it for "console.cloud.google.com", and please update this throughout all affected notebooks.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/217/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='59'/>